slimCat/Utilities/Extensions/StringExtensions.cs

```diff
@@ -216,7 +216,14 @@ public static Gender ParseGender(this string input)
                     return Gender.Cuntboy;
 
                 default: // every other gender is parsed normally
-                    return input.ToEnum<Gender>();
+                    try
```

Comment:
for sanity, it would be better if this was done in the `ToEnum` extension method